src/aolayer.cpp

```diff
@@ -21,6 +21,8 @@ AOLayer::AOLayer(QWidget *p_parent, AOApplication *p_ao_app) : QLabel(p_parent)
   preanim_timer = new QTimer(this);
   preanim_timer->setSingleShot(true);
   connect(preanim_timer, SIGNAL(timeout()), this, SLOT(preanim_done()));
+
+  m_reader = new QImageReader();
```

Comment:
Not destroyed on `AOLayer` destruction. Might want to make `m_reader` a field instead of a pointer, since it is now apparent that `QImageReader` is not a QObject.

Reply:
I _think_ b0796a does this.